spectrome/forward/network_transfer.py

```diff
@@ -3,7 +3,9 @@
 coupling alpha (in laplacian) and the local coupling alpha = 1. """
 
 import numpy as np
-
+import sys
+import matplotlib.pyplot as plt
```

Comment:
Are these imports necessary? I don't see them being used. =/

Reply:
Thanks. I don't know what it went there. I am removing it now.